### PR TITLE
Add --version argument to Assigner

### LINKED ISSUE #109: Put version in help info
T'would be nice to include that information in `help`'s output. We should also include it as a debug log message when Assigner starts so debug output is useful.

### PR DESCRIPTION
**Pull request by LinuxMercedes**
The version is also logged at the DEBUG level.

Version information is now fetched from git tags.

Closes #109

### CODE REVIEW

**Comment by LinuxMercedes** [comment]
I'm not sure that I've done the unit test right here. @brhoades what do you think?

File: setup.py [hunk 1]
```diff
@@ -20,9 +20,8 @@
 setup(
     name='assigner',
 
-    # Versions should comply with PEP 440:
-    # https://www.python.org/dev/peps/pep-0440/
-    version='1.1.0',
+    # Derive version from git tags
+    use_scm_version=True,
```
Comment (brhoades) [inline setup.py]: Neat! This must be a Python 3 thing; I've never seen it.

File: assigner/tests/assigner_test.py [hunk 2]
```diff
@@ -42,8 +42,11 @@ def test_add_default_help(self):
         self.assertTrue(self.mock_parser.set_defaults.called)
         self.assertFalse(self.mock_parser.print_usage.called)
 
+        mock_args = self.mock_parser.parse_args.return_value
```
Comment (brhoades) [inline assigner/tests/assigner_test.py]: +1 for testing, since this is the new default :)

File: assigner/__init__.py [hunk 3]
```diff
@@ -12,6 +12,14 @@
 from assigner.roster_util import get_filtered_roster
 from assigner.progress import Progress
 
+from pkg_resources import get_distribution, DistributionNotFound
+
```
Comment (brhoades) [inline assigner/__init__.py]: np: 2 spaces

Comment (LinuxMercedes) [reply assigner/__init__.py]: Fix'd

**Pull request merged**